app/instrumentation-tests/src/org/commcare/androidTests/FormEntryTest.kt

```diff
@@ -265,6 +277,31 @@ class FormEntryTest: BaseTest() {
                 .check(matches(isDisplayed()))
     }
 
+    @Test
+    @BrowserstackTests
+    fun testQuestionHintVisible() {
+        InstrumentationUtility.login("test_user_7", "123")
+        // Open first form
+        InstrumentationUtility.openForm(0, 0)
+        closeSoftKeyboard()
+
+        //Confirm the hint is visible
+        onView(withClassName(endsWith("ShrinkingTextView")))
```

Comment:
For the sake of completeness, is it possible to also check for the correct text itself.

Reply:
I added the check to verify that the hint text is correct.